assets/css/style.css

```diff
@@ -299,4 +301,95 @@ h1, h2, h3, h4, h5, h6 {
 .fig img {
     display: block;
     transition: opacity .1s linear;
+}
+
+/* Section 2 */
+
+.s2 {
+    display: flex;
+    width: 100%;
+    height: 650px;
+
+    background: #ffffff;    
+    background-size: cover ;
+
+    padding-top: 80px;
+    top: 0%;
+    left: 0%;
+    right: 0%;
+    z-index: 1000;
+}
+
+.s2__inner {
+    width: 100%;
+    max-width: 880px;
+    margin: 0px auto;
+    text-align: center;
+}
+
+.s2__title {
+    text-transform: uppercase;
+    color: #333;
+    font-weight: bold;
+    font-size: 30px;
+    line-height: 1.30px;
+}
+
+.s2__suptitle {
+    font-family: 'Kaushan Script', cursive;
+    font-size: 24px;
+    color: #333;
+    margin-bottom: 30px;
+    font-weight: 400;
+
+}
+
+.s2__title::after {
+    content: "";
+    display: block;
+    width: 60px;
+    height: 3px;
+
+    background-color: #e78080 ;
+    margin: 50px auto 50px;
+}
+
+/* S2_Services */
+
+.services1 {
```

Comment:
This is almost useless class in your code. You can use only border here. Flex isn't working here